### ISSUE TITLE
Change preset playback speed buttons

### ISSUE THREAD

**Issue by Ten-Tails** [TortugaPower/BookPlayer#1171]
## Feature request

I would just like to customize the preset playback speed buttons from their 1x, 2x, and 3x options. Maybe there can be a line in settings that allows you to change how many boxes (from like 3 to 6 maybe) there are and to customize them. 

I typically never play at 3x unless the narrator is really slow and I am reading along with the physical book, but it would be nice to be able to put 1.5x, 1.75x, 2x, etc...

I really love the app, and has been amazing for all my audiobooks!

**Comment by GianniCarlo** [comment]
yes this would be helpful. Implementation details for anyone that wants to help:
- On the Player Controls drawer (presented from the Player screen)
  - Set the X button on the top left corner for dismissing, and replace the Done button with an overflow or More icon
  - This would show an alert sheet with options, the first one would be to customize speed presets (although we can do without this sheet in this first iteration, and just show the new modal screen to set the presets)
  - New drawer (or screen?) with controls to update the three preferences, and we would save the user preferences in UserDefaults
